test_files/tkv_upload_test.json

```diff
@@ -0,0 +1,36 @@
+{
+  "67": {
+    "ncbi_taxonomy_id": 9606,
```

Comment:
I understand what you are trying to do with this now, could you make the key value pairs more generic for the test data set?